### PR TITLE
add more outputs (ffwert,augsburg,freising)

### PR DESCRIPTION
**Pull request by T0biii**
more Outputs for 
https://github.com/freifunkMUC/freifunk.net-API/blob/master/cities_to_nodes_urls.json

### CODE REVIEW

File: yanic/yanic.conf.tmpl [hunk 1]
```diff
@@ -55,6 +55,30 @@ path = "/srv/yanic/meshviewer-ffdon.json"
...
+domain_as_site = true
+sites = [{% for domain in sites["ffwert"] %}"ffwert_{{ domain }}", {% endfor %}]
+
+[[nodes.output.meshviewer-ffrgb]]
+enable = true
+path = "/srv/yanic/meshviewer-augsburg.json"
+
+[nodes.output.meshviewer-ffrgb.filter]
+domain_as_site = true
+sites = "ffmuc_augsburg"
```
Comment (DasSkelett) [inline yanic/yanic.conf.tmpl:72]:
```suggestion
sites = ["ffmuc_augsburg"]
```

File: yanic/yanic.conf.tmpl [hunk 2]
```diff
@@ -55,6 +55,30 @@ path = "/srv/yanic/meshviewer-ffdon.json"
...
+
+[[nodes.output.meshviewer-ffrgb]]
+enable = true
+path = "/srv/yanic/meshviewer-freising.json"
+
+[nodes.output.meshviewer-ffrgb.filter]
+domain_as_site = true
+sites = "ffmuc_freising"
```
Comment (DasSkelett) [inline yanic/yanic.conf.tmpl:80]:
```suggestion
sites = ["ffmuc_freising"]
```

**Review by DasSkelett — CHANGES_REQUESTED** [review]
Sorry for having a look at it only now. For Augsburg and Freising the sites must also be arrays, otherwise it complains. Apart from that, looks good.

**Comment by T0biii** [comment]
atm https://map.ffmuc.net/data/meshviewer-ffdon.json is empty. 
does it have anything to do with this change?

**Review by DasSkelett — APPROVED** [review]
Thanks, this should be good!

**Pull request merged**